substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2930,12 +3009,19 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// Create the main, bonded account of a pool with the given id.
-	pub fn create_bonded_account(id: PoolId) -> T::AccountId {
+	pub fn generate_bonded_account(id: PoolId) -> T::AccountId {
 		T::PalletId::get().into_sub_account_truncating((AccountType::Bonded, id))
 	}
 
+	pub fn migrate_to_delegate_stake(id: PoolId) -> DispatchResult {
```

Comment:
```suggestion
	pub(crate) fn migrate_to_delegate_stake(id: PoolId) -> DispatchResult {
```

Reply:
I use them in the benchmarking crate as well :)